src/TouristInfoCards.js

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+
+const TouristInfoCards =() =>{
+    return (
+    <div class="card-group row">
+        <div class="card">
+            <img
+               src="https://i.postimg.cc/KvdgxZPS/Palace-of-Westminster-from-the-dome-on-Methodist-Central-Hall.jpg"
+               class="card-img-top"
+               alt="London_View"/>
+            <div class="card-body">
+                <h5 class="card-title">London</h5>
+                <p class="card-text">
+                London, the capital of England and the United Kingdom, is a
+                21st-century city with history stretching back to Roman times.
+                At its centre stand the imposing Houses of Parliament, the
+                iconic ‘Big Ben’ clock tower and Westminster Abbey, site of
+                British monarch coronations. Across the Thames River, the
+                London Eye observation wheel provides panoramic views of the
+                South Bank cultural complex, and the entire city.
+                </p>
+                <a href="https://www.visitlondon.com/" class="btn btn-primary">
+                Chekout Upcoming events
+                </a>
+            </div>
+        </div>
+
+        <div class="card">
+            <img
```

Comment:
There's some repeated code in this component. How could we improve this component so that we're not repeating ourselves so much?

Reply:
I figured this will be the best approach considering the way I want to render the page in general